docker-compose.yml

```diff
@@ -1,22 +0,0 @@
-version: '3.1'
-services:
-  postgres:
-    image: postgres:10.2-alpine
-    environment:
-      POSTGRES_USER: ${POSTGRES_USER:-postgres}
-      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-postgres}
-    ports:
-      - "${POSTGRES_PORT:-5432}:5432"
-    volumes:
-      - postgres_data:/var/lib/postgresql/data
-  redis:
-    image: redis:4.0.8-alpine
-    ports:
-      - "${REDIS_PORT:-6379}:6379"
-    volumes:
-      - redis_data:/data
-volumes:
-  postgres_data:
-    driver: local
-  redis_data:
-    driver: local
```

Comment:
Please don't remove the default docker-compose file because it's used for development. If it's impossible to have a default docker compose file and another one please leave this one as is and rename it to sth. like docker-compose.infra.yml

Reply:
The purpose of the change is to use docker-compose for development without the need to run db and app separately. Why `docker-compose.local.yml` cannot be use for development?

Reply:
Because it's unproductive and I don't like it. Docker compose is used to run the databases because it's conveniant. Docker isn't the primary development enviorenement. I don't like rebuilding the image on every change, starting many interactive shells to the docker instance and dealing with production optimised os. I haven't seen anybody use docker like that. If you want feel free to create a docker compose file that suits your particular needs of running the entire application from compose but leave the default one as is.